src/DDTrace/ScopeManager.php

```diff
@@ -33,18 +33,17 @@ public function getActive()
             return null;
         }
 
-        return $this->scopes[count($this->scopes) - 1];
+        return end($this->scopes);
     }
 
     public function deactivate(Scope $scope)
     {
-        $scopeLength = count($this->scopes);
+        $i = array_search($scope, $this->scopes, true);
```

Comment:
We could use `array_search()` to find the index of `$scope` in `$this->scopes`, maybe better than looping?